src/main/java/com/LP2/EventScheduler/controller/ConnectionController.java

```diff
@@ -45,4 +47,20 @@ public ResponseEntity<MessageResponse> acceptInvitation(
                 HttpStatus.OK
         );
     }
+
+
+    @DeleteMapping("/removeConnection/{connectionId}")
+    public ResponseEntity<MessageResponse> removeConnection(
+            @PathVariable("connectionId") UUID connectionId,
+            @RequestAttribute("user") User authUser) {
+        try {
+            MessageResponse response = connectionService.removeConnection(connectionId, authUser);
+            return ResponseEntity.ok(response);
+        } catch (ConnectionNotFoundException e) {
+            return ResponseEntity.status(HttpStatus.NOT_FOUND).body(new MessageResponse("Connection not found"));
+        } catch (IsNotOwnerException e) {
+            return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(new MessageResponse("Unauthorized to remove this connection"));
+        }
```

Comment:
Respecto al manejo de errores debes de saber que no es necesario hacer esto, ya que yo ya he configurado el tema de los errores de que al momento de utilizar un excepción que esta dentro del paquete `exception` este ya lance una respuesta con mensaje y estado de error (puedes ver en los demás endpoints como estos con solo lanzar la excepción es más que suficiente)

Así que cambia eso por esto:
```suggestion
        return ResponseEntity.ok(connectionService.removeConnection(connectionId, authUser));
```